website/app/Http/Controllers/EventsController.php

```diff
@@ -48,9 +51,10 @@ public function store(CreateEventRequest $request)
       $date     = $date === null ? null : new Carbon($date);
       
       Event::create(compact('title', 'date', 'location'));
-      return;
+      return status["Success"];
     }
-    return redirect(action('EventsController@index'));
+    redirect(action('EventsController@index'));
```

Comment:
We can remove the redirect here. Redirect generates a redirect response object that lets laravel know where to send the client, having it here without a return doesn't do anything. Also we are returning JSON now and should never redirect.